app/services/capybara_services/dev_to_parser.rb

```diff
@@ -0,0 +1,24 @@
+require 'capybara/dsl'
+
+module CapybaraServices
+  class DevToParser < ApplicationService
+    def initialize(url)
+      @url = url
+    end
+
+    def call
+      extend Capybara::DSL
```

Comment:
when you include, extend or prepend then you should place it right after class definition